www/js/toolbar.js

```diff
@@ -45,18 +48,64 @@ const toggleOverlayUI = (toolbarItem, show) => {
   });
 };
 
+const setListeners = () => {
```

Comment:
May be: Should move setListeners to a separate file or in share-sync file.

Reply:
Please do keep files small.

Reply:
There's also an `addListener` function in share-sync that handlest the socket check.